pages/model/[uuid].tsx

```diff
@@ -224,6 +229,15 @@ const Model = () => {
                   </ListItemIcon>
                   <ListItemText>Upload new version</ListItemText>
                 </MenuItem>
+                <MenuItem
+                  onClick={requestApprovalReset}
+                  disabled={version?.managerApproved === 'No Response' && version?.reviewerApproved === 'No Response'}
```

Comment:
If this logic starts getting any more complicated we should think about moving it above into an intermediary variable.  It's fine for now, :)